contrib/plugins/plugin-download/download.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/ovh/cds/sdk/plugin"
+)
+
+type DownloadPlugin struct {
+	plugin.Common
+}
+
+func (d DownloadPlugin) Name() string        { return "download" }
+func (d DownloadPlugin) Description() string { return "This is a plugin to download file from URL" }
+func (d DownloadPlugin) Author() string      { return "Benjamin COENEN <benjamin.coenen@corp.ovh.com>" }
+
+// Parameters return parameters description
+func (d DownloadPlugin) Parameters() plugin.Parameters {
+	params := plugin.NewParameters()
+	params.Add("url", plugin.StringParameter, "the url of your file", "{{.cds.app.downloadUrl}}")
+	params.Add("filepath", plugin.StringParameter, "the destination of your file to be copied", ".")
+
```

Comment:
maybe we should manage headers like basic auth headers...
add a "headers param" which should be formatted like "Header-Key-1=VALUE Header-Key-2=VALUE"